concepticondata/conceptlists.tsv

```diff
@@ -309,3 +309,4 @@ Zettersten-2020-24a	Zettersten, Martin and Lupyan, Gary	2020	a	24	naming test, c
 Zettersten-2020-16b	Zettersten, Martin and Lupyan, Gary	2020	b	16	naming test	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the shape stimuli used in experiment 3A and 3B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English. The association values come from [Vanderplas and Garvin (1959)](:bib:Vanderplas1959).		
 Gauchat-1925-480	Gauchat, Louis and Jeanjaquet, Jules and Tappolet, Ernest	1925		480	questionnaire	French	French dialects of Switzerland		Gauchat1925		This questionnaire was used to study differences among French dialects of Switzerland. It consists of 480 items which often can be taken together to form phrases. Our translation has tried to reflect this in so far as we have not translated the items point-by-point, but rather tried to determine the overall meaning of the intended phrase first. The concept mapping is sporadic and follows a couple of rules. We first mapped all nouns that have a direct article and appear in singular. In a second run, we tried to map missing nouns that had not been mapped, which appear with an indirect article or in plural or other forms. The same was done for the verbs, giving preference where possible to a clear infinitive, rather than inflected verb forms.	2-159	
 Ratcliffe-2020-100	Ratcliffe, Robert R.	2020		100	areal, questionnaire	English	Arabic dialects	https://doi.org/10.1163/22105832-01001100	Ratcliffe2020		This concept list is supposed to reflect [Swadesh's list of 100 items](:ref:Swadesh-1955-100) and was used for a lexicostatistic study of Arabic dialects. The mappings may differ in some cases from the original list of Swadesh, as we had problems to identify some meanings unambiguously from the glosses (for RAIN, for example, no further specification was given). The list was provided in WORD format in the supplementary material of the study and turned out to be corrupted by space characters inserted inside almost all words including the numbers. These spaces were deliberately deleted in our version, to allow for a convenient inspection of the list.	1-29	
+Davletshin-2012-100	Davletshin, Albert	2012		100	basic, areal, proto-language	English	Uto-Aztecan languages	http://www.jolr.ru/files/(95)jlr2012-8(75-92).pdf	Davletshin2012		The concepts of this list were taken from [Swadesh's 100-item list](:ref:Swadesh-1955-100) and formed the basis for comparing 9 different Uto-Aztec languages. Based on this comparison, the author proposed possible corresponding forms of Proto-Aztecan.		
```

Comment:
Maybe paraphrase the first sentence a bit? I suggest: `The concepts for this list were taken from [Swadesh's 100-item list](:ref:Swadesh-1955-100), which formed the basis for comparing 9 different Uto-Aztecan languages.`

It's still not clear to me which lists are to be tagged as 'areal'. If this tag describes lists that were used for specific languages/a specific geographic area, then almost every list on Concepticon (with some minor exceptions) should be tagged as such. I was convinced this tag is used for those lists only that differ from basic ones and include a significant share of 'areal' concepts (e.g., certain plants or cultural artifact, typical of this area/languages).